### ISSUE TITLE
Windows 10 installation ISO on x86_64-efi

### ISSUE THREAD

**Issue by parkerlreed** [a1ive/grub2-filemanager#264]
I have the latest Windows 10 ISO (2004) on an exFAT partition. 

Boot Windows option loads the installer, I can go through the partitioning menu, but as soon as the installer starts it crashes with a general error. 

Boot Map loads up until the the first menu after clicking the first Install button and then complains it has not disk drivers and can't see any fixed disks. Throws up the menu for selecting a driver to load and you can't get past it at all. 

Is there something I'm missing here?

EFI is on partition 1 and the exFAT is partition 4. 

![unknown-20-1](https://user-images.githubusercontent.com/841440/94345274-bad65a80-fff2-11ea-8f03-a00f3319e906.png)

**Comment by steve6375** [comment]
The map option works as expected - it will not work.
The Windows option should not crash - please give details of the 'general error' - picture if possible.

**Comment by a1ive** [comment]
Are you trying to install the system on the hard drive where the Windows ISO is located? It can't be done.You need another hard drive to store the ISO.

**Comment by parkerlreed** [comment]
It was to a separate fixed HDD. I just tried to recreate this in VirtualBox and it's working. May have been a weird quirk with the hardware I was booting on (some random Lenovo tower, fairly recent). I can check back with that computer beginning of next week.

**Comment by parkerlreed** [comment]
"Windows could not prepare the computer to boot into the next phase of installation. To install Windows, restart the installation."

I get this error after clearing out all the partitions on the fixed HDD and starting the installation process. 

https://youtu.be/FcLXN5myfHo

**Comment by steve6375** [comment]
Explain disk setup? USB boot disk? Internal disks?
What disk is the BIOS set to boot to?
Is the internal disk a known good disk?

**Comment by parkerlreed** [comment]
USB boot disk loading to rEFInd first, which then loads GrubFM

Manually booted the USB from the UEFI boot menu on the computer.

Internal disk is good. 500GB spinning rust. I was able to do a Windows install from a normal USB drive (created from Media Creation Tool) just fine.

Lenovo V520s

**Comment by steve6375** [comment]
I am not sure of the cause.
Maybe you could look at the log files?
When you get the error - type SHIFT+F10 to get to cmd console of Windows
Then type dir /ad /s X:\WINDOWS\*.log
and use Notepad to look at the log files.

**Comment by steve6375** [comment]
This is just a guess. The problem may be that you are booting from a hard disk which has a GPT EFI boot partition and Windows thinks that the USB EFI boot partition is the system boot partition (not sure why?) but then realises it is an external USB disk and so refuses to install the boot code/alter the BCD. The log files should give you a clue.
So maybe your USB boot disk needs to just have a simple FAT32 boot partition or just MBR\Legacy partitions?

**Comment by parkerlreed** [comment]
Yeah that's what I was starting to think as well. I will check the logs when I get a chance.
